components/kserve/kserve.go

```diff
@@ -16,7 +16,7 @@ import (
 
 const (
 	ComponentName          = "kserve"
-	Path                   = deploy.DefaultManifestPath + "/" + ComponentName + "/base"
+	Path                   = deploy.DefaultManifestPath + "/" + ComponentName
```

Comment:
Why is this change required?

Reply:
https://github.com/opendatahub-io/kserve/tree/master/config does not have a "`base`" folder.

i will need config get_all_manifests.sh to use 
`config/overlays/odh` as path